_layouts/product-overview.html.slim

```diff
@@ -31,9 +31,12 @@ h2#overview Overview
 
   .large-11.columns.product-video
     = video_player(video(page.product.intro_video)) if page.product.intro_video
+    - if page.product.intro_youtube
+      = video_player(site.videos["#{page.product.intro_youtube}"])
```

Comment:
Can you match the above syntax? It's hard to read with the mixture. E.g:

        = video_player(site.videos["#{page.product.intro_youtube}"]) if page.product.intro_youtube

Also, would this work?

       = video_player(video(page.product.intro_youtube)) if page.product.intro_youtube